packages/lambda-powertools-middleware-correlation-ids/__tests__/index.js

```diff
@@ -457,5 +498,7 @@ describe('Correlation IDs middleware', () => {
 
   describe('SQS', () => sqsTests())
 
+  describe('SQS Without Raw Message', () => sqsWithoutRawDeliveryTests())
```

Comment:
I'm probably just nick-picking here, but I'm not fond of the `SQS without raw message` name as the distinction of raw message or not is specific to SNS => SQS. Can we try a more meaningful name? e.g. "wrapped SNS message", "nested SNS message", "SNS message"?

Reply:
test names now updated to `SQS wrapped SNS message`